heat/examples/vm_two_nics_with_share.yaml

```diff
@@ -74,17 +74,7 @@ resources:
             key_name: { get_param: user_key }
             image: { get_param: user_vm_image }
             flavor: { get_param: user_vm_flavor }
-            user_data: |
-                #!/usr/bin/env python
-                import json,re,requests
-                r = requests.get('http://169.254.169.254/openstack/latest/meta_data.json')
-                data = r.json()
-                m = re.search('\'(.+?)\'', data['meta']['share_export_loc_path'])
-                sharepath = m.group(1)
-                mountpoint = '/mnt/share'
-                os.mkdir(mountpoint)
-                os.chown(mountpoint,0,0)
-                os.system("mount %s %s" % (sharepath,mountpoint))
+            user_data: { get_file: 'https://raw.githubusercontent.com/hpcugent/openstack-templates/HeatTemplateWithShare/heat/examples/mount_share.py' }
```

Comment:
This URL is still not correct, `HeatTemplateWithShare` branch is not available from upstream, it should be changed by the master branch. Also move the script to a different directory like `heat/scripts//mount_share.py`.
So the URL should look like this:

` user_data: { get_file: 'https://raw.githubusercontent.com/hpcugent/openstack-templates/master/heat/scripts/mount_share.py' }`

If you move the scripts into ../heat/scripts folder.